### PR TITLE
Feat. improved inline link styles, and text input focus when inverted

### LINKED ISSUE #243: Inline link states
## Is your feature request related to a problem? Please describe.

Currently the active state of inline links is running into the text surrounding it.

## Describe the solution you'd like

Screen Shot 2020-02-13 at 10 22 08 AM

Main changes include:
- Trimming down the focus state to be extremely close to the text
- Reducing the active state double outline to 1px instead of 2

### PR DESCRIPTION
**Pull request by jhung**
* [x] I've read the [guidelines for Contributing to the Platform Co-op Toolkit](https://github.com/platform-coop-toolkit/.github/blob/master/CONTRIBUTING.md)
* [x] This isn't a duplicate of an existing pull request

## Description

Fixed inline link styles to match designs. Fixes #243.
Fixed focus style for text input for inverted colours so focus border appears outset instead of inset (matching the default non-inverted style). 

## Steps to test

**Inline Link Style**
1. Go to: https://deploy-preview-358--pinecone.netlify.app/index.html
  * Link > Default (Inverse)
  * Link > External (Inverse)
  * Link > Relative (Inverse)
2. Confirm that focus, hover, and active states appear as described in #243 for inline links.

**Text Input Focus Style**
1. Go to: https://deploy-preview-358--pinecone.netlify.app/index.html
  * Form Elements > Text Input > Default (Inverse)
  * Form Elements > Text Areas > Default (Inverse)
  * Form Elements > Selects > Default (Inverse)
2. Confirm that focus style appears outside of the form element.

## Related issues

Fixes #243 .

### CODE REVIEW

**Comment by jhung** [comment]
Looking into the conflict.

**Comment by cherylhjli** [comment]
I'm not seeing the hover state underline colour change - The colour change should be to blue 300. other than that it looks great.

**Comment by greatislander** [comment]
@cherylhjli It's not supported on Chrome, which might be why.

**Comment by cherylhjli** [comment]
I'm not seeing it on firefox and safari either? I see that it's a lighter blue - blue 200

**Comment by greatislander** [comment]
Okay, maybe we are talking about something different— probably easier for you and @jhung to figure it out!

**Comment by jhung** [comment]
@greatislander and @cherylhjli ready for another look.

**Comment by cherylhjli** [comment]
The hover looks great! i'm just realizing now - the standalone links have reversed active and focus style from each other (focus = reverse color, active = just border). Could we make that consistent w/ the standalone links?

**Comment by jhung** [comment]
> The hover looks great! i'm just realizing now - the standalone links have reversed active and focus style from each other (focus = reverse color, active = just border). Could we make that consistent w/ the standalone links?

@cherylhjli I wonder if it should be the other way - the standalone links should be updated to match how it is now for other interactive elements:
* Border = focus
* Border + Background color change = activation
* Color change = hover

Examples:
* [Borderless button](https://pinecone.netlify.app/components/detail/button--borderless.html)
* [Toggle Button](https://pinecone.netlify.app/components/detail/button--toggle-button.html)

**Comment by greatislander** [comment]
@cherylhjli Just following up on this — any thoughts on @jhung's last comment?

**Comment by jhung** [comment]
@greatislander  - @cherylhjli and I talked about it and it's all sorted out. The stand-alone links need to be updated to follow the new scheme. I'm working on that currently, but in the process reveals some needed refactoring of styles.

File: src/assets/styles/components/_link.scss [hunk 1]
```diff
@@ -159,6 +160,55 @@ a:not([class]),
 	--focus-color: var(--white);
 }
 
+ul a:not([class]),
```
Comment (greatislander) [inline src/assets/styles/components/_link.scss:163]:
```suggestion
ul:not(.link-list) a:not([class]), 
ol a:not([class]),
blockquote a:not([class]),
```

**Pull request merged**